File: app/src/main/java/com/friendlypos/distribucion/modelo/TipoProducto.java
package com.friendlypos.distribucion.modelo;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by Delvo on 04/11/2017.
 */

public class TipoProducto extends RealmObject {

   /* {
        "id": 1,
            "name": "Gravado",
            "created_at": "-0001-11-30 00:00:00",
            "updated_at": "-0001-11-30 00:00:00"
    }*/

    @PrimaryKey
    String id;

    String name;
    String created_at;
    String updated_at;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCreated_at() {
        return created_at;
    }

    public void setCreated_at(String created_at) {
        this.created_at = created_at;
    }

    public String getUpdated_at() {
        return updated_at;
    }

    public void setUpdated_at(String updated_at) {
        this.updated_at = updated_at;
    }

    @Override
    public String toString() {
        return "TipoProducto{" +
                "id='" + id + '\'' +
                ", name='" + name + '\'' +
                ", created_at='" + created_at + '\'' +
                ", updated_at='" + updated_at + '\'' +
                '}';
    }
}

File: app/src/main/java/com/friendlypos/distribucion/modelo/TipoProducto.kt
package com.friendlysystemgroup.friendlypos.distribucion.modelo

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

/**
 * Created by Delvo on 04/11/2017.
 */
open class TipoProducto : RealmObject() {
    /* {
            "id": 1,
                "name": "Gravado",
                "created_at": "-0001-11-30 00:00:00",
                "updated_at": "-0001-11-30 00:00:00"
        }*/
    @PrimaryKey
    var id: String? = null

    @JvmField
    var name: String? = null
    var created_at: String? = null
    var updated_at: String? = null

    override fun toString(): String {
        return "TipoProducto{" +
                "id='" + id + '\'' +
                ", name='" + name + '\'' +
                ", created_at='" + created_at + '\'' +
                ", updated_at='" + updated_at + '\'' +
                '}'
    }
}
